Models

barld opened the pull request:
create database model

barld committed:
Merge remote-tracking branch 'origin/develop' into core-2.1

barld committed:
first version of database models

barld commented:
Need review and modifications

barld committed:
add doxyfile

barld committed:
Merge remote-tracking branch 'origin/core-2.1' into models

barld committed:
add required atributes

wesselmeijer reviewed (dismissed):
Een paar engelse spelfouten in de comments maar qua code ziet het er goed uit

barld committed:
Add extra validation and properties

barld committed:
Merge branch 'develop' into models

barld committed:
first model with good migration

barld commented:
De eerste versie waarvan werkelijk een succesvolle migration is te maken staat er op.

iksman commented on GoedeDoelenHelpen/Data/ApplicationDbContext.cs:
```diff
@@ -1,16 +1,190 @@
+        public Charity Charity { get; set; }
+    }
+
+    public class EventTeam
```
EventTeam moet volgens de ERD een relatie met ApplicationUser hebben, deze bestaat echter niet in de code

wesselmeijer commented on GoedeDoelenHelpen/Data/ApplicationDbContext.cs:
```diff
@@ -1,16 +1,190 @@
+        public string Name { get; set; }
+    }
+
+    public class EventParticipant
```
EventParticipant moet een relatie hebben met ApplicationUser volgens de ERD, deze ontbreekt echter.

wesselmeijer committed:
Added relation and checked spelling

Added relation between EventP. and ApplicationUser.
Also fixed several spelling errors.

barld committed:
move models to sepretate files

wesselmeijer committed:
Fixed + and Added Migration

Fixed weird GitHub + in code also added migration

fjwillemsen reviewed (approved):
This pull request checks out, issues have been resolved.
